extensions/amp-next-page/1.0/page.js

```diff
@@ -115,6 +145,12 @@ export class Page {
     if (visibilityState == this.visibilityState_) {
       return;
     }
+
+    //Reload the page if necessary
+    if (this.isPaused() && visibilityState === VisibilityState.VISIBLE) {
```

Comment:
Do you also need to update `this.visibilityState_` here?

Reply:
Hmm it does gets updates on line 155 (this does not return yet)